web/tests/functional/db_cleanup/test_db_cleanup.py

```diff
@@ -211,12 +211,13 @@ def test_garbage_file_collection(self):
         self._cc_client = env.setup_viewer_client(self.test_workspace)
         self.assertIsNotNone(self._cc_client)
 
-        self.assertEqual(len(files_in_report_before & files_in_report_after),
-                         0)
+        self.assertNotEqual(
```

Comment:
Why was the source file garbage collection behavior modified here?
If some source file was modified those source files should be removed and new source files should be added stored to that report?
Shouldn't be that tested here? However the test function name doesn't say much `test_garbage_file_collection`.
Should we check if the file ids are not the same before and after and the file ids which belong to the before part are not available anymore?